src/platforms/stm32/src/lib/sys.c

```diff
@@ -18,15 +18,17 @@
  * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
  */
 
-#include <sys.h>
-
 #include <avmpack.h>
 #include <defaultatoms.h>
 #include <scheduler.h>
+#include <sys.h>
+
+#include <trace.h>
 
+#include "avm_log.h"
 #include "gpiodriver.h"
 
-#include "trace.h"
+#define TAG "System"
```

Comment:
I don’t have a strong opinion, by maybe use “AtomVM” here, too?

Reply:
I don't have a strong opinion either, I went back and forth between "AtomVM" and "System" for the tag... The only reason I ended up using "System" was to help identify the source of the message quicker if line info is turned off... but I tend to agree that "AtomVM" does seem preferable for the TAG.